README.md

```diff
@@ -1093,8 +1093,41 @@ The ECS CLI supports creating, running, inspecting and stopping tasks defined by
 #### ecs-cli local up
 
 #### ecs-cli local ps
+Once you have your task running locally, the basic command to list your task's containers is:
+ ```
+$ ecs-cli local ps
+```
+This will search for containers created from the `./task-definition.json` file (to see all available options, run `ecs-cli local ps --help`).  
+
+For example, if you'd like to list containers created from another task definition file, use the following command:
```

Comment:
Maybe instead of "from another task definition file" you can say "from a specific task definition file"

Reply:
Done!